lib/ImportStatements.js

```diff
@@ -9,13 +9,15 @@ import uniqBy from 'lodash.uniqby';
 import Configuration from './Configuration';
```

Comment:
One minor nit: if it isn't too much trouble, could you configure your editor to wrap git commit messages at 72 characters?